Annex60/Fluid/Movers/FlowControlled_m_flow.mo

```diff
@@ -20,7 +20,7 @@ model FlowControlled_m_flow
   parameter Modelica.SIunits.MassFlowRate m_flow_start(min=0)=0
     "Initial value of mass flow rate"
     annotation(Dialog(tab="Dynamics", group="Filtered speed"));
-
+    // fixme: shouldn't the massFlowRates not also come from the performance record?
```

Comment:
This still needs to be sorted out. I would not involve the performance records in this parameter.